client/app/dedicatedCloud/datacenter/drp/index.js

```diff
@@ -1,47 +1,55 @@
 import {
+  DEDICATEDCLOUD_DATACENTER_DRP_IP_BLOCK_REG_EXP,
   DEDICATEDCLOUD_DATACENTER_DRP_IP_USAGE_MAC_ADDRESS_REG_EXP,
   DEDICATEDCLOUD_DATACENTER_DRP_OPTIONS,
   DEDICATEDCLOUD_DATACENTER_DRP_ORDER_OPTIONS,
   DEDICATEDCLOUD_DATACENTER_DRP_ROLES,
   DEDICATEDCLOUD_DATACENTER_DRP_STATUS,
   DEDICATEDCLOUD_DATACENTER_DRP_UNAVAILABLE_IP_STATUS,
+  DEDICATEDCLOUD_DATACENTER_DRP_VPN_CONFIGURATION_STATUS,
   DEDICATEDCLOUD_DATACENTER_ZERTO,
 } from './dedicatedCloud-datacenter-drp.constants';
 
+import choiceTemplate from './configuration/dedicatedCloud-datacenter-drp-choice.html';
 import component from './dedicatedCloud-datacenter-drp.component';
 import controller from './dedicatedCloud-datacenter-drp.controller';
-import confirmationStep from './steps/ovh/confirmation';
-import mainPccStep from './steps/ovh/mainPcc';
-import secondPccStep from './steps/ovh/secondPcc';
+import onPremiseTypeConfiguration from './configuration/steps/onPremise';
+import ovhTypeConfiguration from './configuration/steps/ovh';
 import service from './dedicatedCloud-datacenter-drp.service';
+import summary from './summary';
 
 const componentName = 'dedicatedCloudDatacenterDrp';
 const controllerName = 'DedicatedCloudDatacenterDrpCtrl';
 const moduleName = 'dedicatedCloudDatacenterDrp';
-const serviceName = 'DedicatedCloudDrp';
+const serviceName = 'dedicatedCloudDrp';
 
+const ipBlockReqExpConstantName = 'DEDICATEDCLOUD_DATACENTER_DRP_IP_BLOCK_REG_EXP';
 const ipUsageMacAddressRegExpConstantName = 'DEDICATEDCLOUD_DATACENTER_DRP_IP_USAGE_MAC_ADDRESS_REG_EXP';
 const optionsConstantName = 'DEDICATEDCLOUD_DATACENTER_DRP_OPTIONS';
 const orderOptionsConstantName = 'DEDICATEDCLOUD_DATACENTER_DRP_ORDER_OPTIONS';
 const rolesConstantName = 'DEDICATEDCLOUD_DATACENTER_DRP_ROLES';
 const statusConstantName = 'DEDICATEDCLOUD_DATACENTER_DRP_STATUS';
 const unavailableIpStatusConstantName = 'DEDICATEDCLOUD_DATACENTER_DRP_UNAVAILABLE_IP_STATUS';
+const vpnConfigurationStatusConstantName = 'DEDICATEDCLOUD_DATACENTER_DRP_VPN_CONFIGURATION_STATUS';
 const zertoConstantName = 'DEDICATEDCLOUD_DATACENTER_ZERTO';
 
 angular
   .module(moduleName, [
-    confirmationStep,
-    mainPccStep,
-    secondPccStep,
+    onPremiseTypeConfiguration,
+    ovhTypeConfiguration,
+    summary,
   ])
   .component(componentName, component)
+  .constant(ipBlockReqExpConstantName, DEDICATEDCLOUD_DATACENTER_DRP_IP_BLOCK_REG_EXP)
```

Comment:
If `constants` are not used outside of module, there might be no need to expose it from an `AngularJS` point of view

Reply:
Good point